test/test_facebook.py

```diff
@@ -197,5 +197,40 @@ def test_parse_signed_request_when_correct(self):
         self.assertTrue('algorithm' in result)
 
 
+class TestAPIRequest(FacebookTestCase):
+    def test_request_args_by_none(self):
+        FB_VER = 2.7
+        FB_OBJECT_ID = "1846089248954071_1870020306560965"
+        token = facebook.GraphAPI().get_app_access_token(
+            self.app_id, self.secret)
+        graph = facebook.GraphAPI(access_token=token, version=FB_VER)
+
+        # args=None
+        self.assertRaises(
+            TypeError, graph.request, FB_OBJECT_ID, args=None, post_args=None)
+
+    def test_request_args_in_dict(self):
```

Comment:
Is this test strictly necessary? The only scenario I can think of where this test fails and `test_request_args_by_default` succeeds is the one I raised in a previous comment - the structure of the request method changes so the default value of `args` is no longer an empty dict.